frontend/src/routes/WildcardRoute.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Navigate, useSearchParams } from 'react-router-dom-v5-compat';
+
+import * as AlertUtils from '../utils/AlertUtils';
+
+const OVERVIEW_ROUTE = '/overview';
+
+export const WildcardRoute = (): JSX.Element => {
+  const [searchParams] = useSearchParams();
+  const openshiftError = searchParams.get('openshift_error');
+  const cluster = searchParams.get('cluster');
+
+  if (openshiftError) {
+    const clusterMessage = cluster ? `to cluster ${cluster}` : 'to cluster';
```

Comment:
nit: Quotes may make the name more obvious
```suggestion
    const clusterMessage = cluster ? `to cluster "${cluster}`" : 'to cluster';
```